packages/example-project/sketches/solid/index.ts

```diff
@@ -39,6 +40,8 @@ export default class Solid {
     this.root.rotation.y += params.rotSpeedY * baseSpeed * deltaFrame
     this.root.rotation.z += params.rotSpeedZ * baseSpeed * deltaFrame
 
+    this.root.position.set(...params.position)
```

Comment:
Here the array format works perfectly for three js, having it as object with `x`, `y` and `z` properties would just be more work, but I can imagine other times it would be nicer to have it the other way around. However for most cases, including color, three.js plays nicely with plain array so maybe we stick with the simplest approach.